patches/sonic-pi-daemon.patch

```diff
@@ -0,0 +1,82 @@
+diff --git a/app/server/ruby/bin/daemon.rb b/app/server/ruby/bin/daemon.rb
+index c302e9fc4..f5807517e 100755
+--- a/app/server/ruby/bin/daemon.rb
++++ b/app/server/ruby/bin/daemon.rb
+@@ -1216,62 +1216,27 @@ module SonicPi
+       end
+ 
+       def run_pre_start_commands
+-        case Util.os
+-        when :linux, :raspberry
+-          #Start Jack if not already running
+-          if `jack_wait -c`.include? 'not running'
+-            #Jack not running - start a new instance
+-            Util.log "Jackd not running on system. Starting..."
+-            @jack_booter = JackBooter.new
+-          else
+-            Util.log "Jackd already running. Not starting another server..."
+-          end
+-        end
++        # do nothing, no need to start jack server
```

Comment:
Note that even without the direct pipewire support, this would be fine, and actually might be source of the previous problem.